src/Microsoft.Fx.Portability/ObjectModel/NuGetPackageInfo.cs

```diff
@@ -35,8 +45,8 @@ public override bool Equals(object obj)
             if (obj is NuGetPackageInfo other)
             {
                 return string.Equals(other.AssemblyInfo, AssemblyInfo, StringComparison.Ordinal)
-                    && Target.Equals(other.Target)
-                    && SupportedPackages.SequenceEqual(other.SupportedPackages);
+                    && string.Equals(other.PackageId, PackageId, StringComparison.Ordinal)
+                    && SupportedVersions.SequenceEqual(other.SupportedVersions);
```

Comment:
The `SupportedVersions` comparison should probably have a comparer

Reply:
Added comparer.